subsys/net/lib/lwm2m/Kconfig

```diff
@@ -122,6 +122,7 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	bool "Firmware Update object pull support"
 	default y
 	depends on LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT
+	select HTTP_PARSER_URL_ONLY
```

Comment:
I did add "select HTTP_PARSER" at [subsys/net/lib/http/Kconfig#L68](https://github.com/rbtchc/zephyr/blob/ee49c3dbdfff43f3ecd0183cdeeffcc1291a0d1c/subsys/net/lib/http/Kconfig#L68) when config HTTP_PARSER_URL_ONLY is selected.
In this way, when LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT is selected, both HTTP_PARSER & HTTP_PARSER_URL_ONLY will be selected as well.